test/test-viz.cpp

```diff
@@ -0,0 +1,350 @@
+#define SDL_MAIN_HANDLED
+
+//Standard libs includes
+#include <iostream>
+#include <vector>
+#include <string>
+#include <tuple>
+#include <variant>
+
+//External libs includes
+#include <lorina/bench.hpp>
+#include <SDL.h>
+
+#define SCREEN_WIDTH 1920
+#define SCREEN_HEIGHT 1080
+
+#define HOR_SPACING 2
+#define VER_SPACING 2
+
```

Comment:
Non-configurable Integer constants must be enumerations, e.g. 
```suggestion
enum {
    HSPACING = 2,
    VSPACING = 2
}
```